ui/src/components/Publication/SimpleResult/index.tsx

```diff
@@ -19,142 +19,165 @@ const SimpleResult: React.FC<Props> = (props): React.ReactElement => {
     const router = useRouter();
     const [error, setError] = React.useState('');
```

Comment:
I see this error state is not being used anymore, but it's still being set from try/catch blocks. What happens if the user clicks on 'Create Draft Version' and the API returns an error? Also, if this state it's not needed, we should remove it completely.

Reply:
Doesn't look like it was ever specced, but it makes sense to show an error message if the new draft creation fails. I've added an alert in that case